src/Resources/views/standard_layout.html.twig

```diff
@@ -113,7 +115,7 @@ file that was distributed with this source code.
     </head>
     <body
             {% block body_attributes -%}
-                class="sonata-bc skin-black fixed
+                class="sonata-bc {{ _skin }} fixed
```

Comment:
```suggestion
                class="sonata-bc {% block adminlte_2_skin_class %}skin-black{% endblock %} fixed
```
Only this change is require to add this PR feature. We shouldn't add `skin` option becouse we working on add support for other templates.

Reply:
I think all the css are not loaded by default, that's why the option is needed.

Reply:
It is not problem. People can add css by `sonata_admin.assets.extra_stylesheets`.

Reply:
IMO we already have the code to remove stylesheets and add new ones built in. So adding a block seems the best option to me.

Reply:
Maybe some doc should be added then.

How to use another LTE skin:
- Remove black stylesheet
- Add you favorite stylesheet
- Override the layout file

Reply:
@wbloszyk that'd be another option. The thing is, that'd you'd still need to override a template block and you'd need to remove and add the template(s) manually. The option I introduced will do that automatically based on the available Admin LTE skins.

I think introducing this block is still a good idea but then I'd name it differently (e.g. `sonata_body_class`).

@VincentLanglet I think in your case a block like this would be very helpful so you're able to switch skins more easily based on the user role.

Reply:
@fastnloud
Lets create "sonata.admin.template.skin_selector" or something simmilar. Then skin can be return by people preference(time, roles ...), can be validate by used template(AdminLTE2, AdminLTE3, custom)... Then new option like `templete_skin_service` can be add. `skin` option is not related to AdminBundle but to AdminLTE2.

Reply:
IMO that would bind php to css and would be bad for when we want to integrate webpack. There is a way currently in the code to replace javascript or css, and we should use that, the only need is to be able to override the main class, which is easy to do in twig with a simple block, no need to introduce complex logics

Reply:
when we switch to webpack encore with admin LTE 3 things wil work different anyway because then you could override the sass variables and create your own skin. But I also think we should improve Sonata 3.x and it's usability to switch skins. An skin option like this is the most easy way to consume and would be great for the time being until we reach sonata 4.x.
So my vote would be to add the logic that is proposed in this PR but also add the twig block to allow more complex logic like @VincentLanglet showed us. By adding the block you don't have to override the complete body_attributes which contains other logic as well. Also a readme with how skin switching works helps adapting to these changes. For most people nothing changes because the default skin is black anyway.

We have settings for using select2, confirms etc. so it would be strange in my opnion to not have a skin setting and let the user remove / add stylesheets themselves.